osrf/vrx
# [Garden] Port light buoy plugin

This pull request ports the `LightBuoyPlugin` from Gazebo classic. I added a new `scan_dock_deliver_task.sdf` world for testing. 

### How to test it?

Launch the `scan_dock_deliver_task` world.

You should observe the light buoy floating and with the `red`->`green`->`blue`->`off` sequence enabled.

![light_buoy](https://user-images.githubusercontent.com/1440739/198642841-975db6f1-68b6-4d38-97ce-7618c97b5258.gif)
